### ISSUE TITLE
CF Perks Fulfillment 

### ISSUE THREAD

**Issue by heiwanalady** [creativedisturbance/public-website#81]
T-shirt and sticker designs are completed.
Currently waiting on people to respond to the survey with their t-shirt size and shipping address. 

people who haven't responded to survey:
people who have:

**Comment by heiwanalady** [comment]
Just got a full cost estimate from the vendor today! Waiting for Roger's approval.

**Comment by porteriffic** [comment]
Already have stickers and t-shirts. Will work on putting together packages starting end of next week. Stephanie needs to talk to Nancy/Charlotte about mailing packages internationally. Will have cost estimate for final packaging in a week or two.

**Comment by heiwanalady** [comment]
This task has been 90% completed. T-Shirts and Stickers have been sent to backers who responded to survey in mid June. 

New Task is for @linamoon to make sure any "straggler" backers (those who answer the survey late) receives their perks. 
I have already discussed with her.

**Comment by halclark** [comment]
This issue was moved to creativedisturbance/podcasts#179
